compiler/modules/print.llvm/llvm.bal

```diff
@@ -200,6 +201,23 @@ public class Module {
         self.context = context;
     }
 
+    // Corresponds to LLVMCreateDIBuilder
+    public function createDIBuilder() returns DIBuilder {
+        DIBuilder dIBuilder = new(self);
+        self.dIBuilder = dIBuilder;
+        return dIBuilder;
+    }
+
+    // Corresponds to LLVMAddModuleFlag
+    public function addModuleFlag(ModuleFlagBehavior behavior, ModuleFlag flag) {
+        DIBuilder? dIBuilder = self.dIBuilder;
```

Comment:
This doesn't feel like a good design.

Reply:
Changed such that module will support multiple `DIBuilders` and will create a new one silently if none available.

Reply:
Still doesn't look good to me. Why does addModuleFlag need a DIBuilder?

Reply:
Fixed